zookeeper/config.sls

```diff
@@ -8,16 +8,16 @@
 zookeeper-data-dir:
   file.directory:
     - name: {{ zk.data_dir }}
-    - user: zookeeper
-    - group: zookeeper
+    - user: {{ zk.user }}
+    - group: {{ zk.group }}
     - makedirs: True
 
 zoo-cfg:
   file.managed:
     - name: {{ zk.real_config }}/zoo.cfg
     - source: salt://zookeeper/conf/zoo.cfg
-    - user: root
-    - group: root
+    - user: {{ zk.user }}
+    - group: {{ zk.group }}
```

Comment:
Having writable directories owned by `zk.user` is definitely a mandatory thing, but why the zookeeper process would need to write into its own config file?
I think it would be better to leave it owned by `root`, or restrict permissions.

There are few files below where the same question applies.

Reply:
Hi @vutny,

thanks for your feedback, let me look into it.

Reply:
Hi @vutny,

ok, agree with you. It is not necessary that zk user own zookeeper-env.sh or zoo.cfg. I have reverted my last commit:

70608029dc36a02f1b6ba97c5022ad37b05fcda9 - change owner of zookeeper-env.sh and zoo.cfg to zk.user and zk.group